source/sync/go-to-production/production-checklist.txt

```diff
@@ -62,9 +62,9 @@ Atlas oplog
 
 App Services schema data consistency
   If you create or modify Device Sync documents using another tool, such as the
-  :binary:`~bin.mongo` shell or MongoDB Compass, be sure the documents validate 
-  against the App Services Schema for the collection. For more information, see 
-  :ref:`Unsynced Documents <unsynced-documents>`.
+  `mongosh shell <https://www.mongodb.com/docs/mongodb-shell/>`__ or MongoDB Compass, 
```

Comment:
should all be `mongosh shell`? i'm seeing `mongo shell` also in updates in this pr